app/components/contributor-list/component.js

```diff
@@ -1,21 +1,46 @@
 import { filterBy } from '@ember/object/computed';
-import { computed } from '@ember/object';
 import Component from '@ember/component';
-import { loadRelation } from 'ember-osf/utils/load-relationship';
```

Comment:
The only remaining use of `loadRelation` is in `preprint-file-browser`, should we remove that too?

Reply:
Should probably remove it during [MOD-176](https://openscience.atlassian.net/browse/MOD-176) that moves queryHasMany to ember-osf.